### ISSUE TITLE
UI breaks when logo src fails and alt text is used

### ISSUE THREAD

**Issue by Senthamizhselven** [FOSSUChennai/Communities#445]
<img width="684" height="439" alt="Image" src="https://github.com/user-attachments/assets/92308ee1-4fe0-48ab-a52a-95559f4d7ae1" />

<img width="403" height="35" alt="Image" src="https://github.com/user-attachments/assets/ffe9e681-7140-4f79-ba35-375cd5a6e494" />
